infra/app/env-template/main.tf

```
@@ -41,32 +59,61 @@ module "service_cluster" {
   cluster_name = local.cluster_name
 }
 
-module "participant_portal" {
-  source                = "../../modules/service"
-  service_name          = "${local.project_name}-participant-${var.environment_name}"
-  image_repository_name = module.app_config.image_repository_name
-  image_tag             = var.image_tag
-  vpc_id                = data.aws_vpc.default.id
-  subnet_ids            = data.aws_subnets.default.ids
-  service_cluster_arn   = module.service_cluster.service_cluster_arn
+module "participant" {
+  source               = "../../modules/service"
+  service_name         = local.participant_service_name
+  image_repository_url = data.aws_ecr_repository.participant_image_repository.repository_url
+  image_repository_arn = data.aws_ecr_repository.participant_image_repository.arn
+  image_tag            = var.image_tag
+  vpc_id               = data.aws_vpc.default.id
+  subnet_ids           = data.aws_subnets.default.ids
+  service_cluster_arn  = module.service_cluster.service_cluster_arn
+  container_port       = 3000
+  container_secrets = [
+    {
+      name      = "DATABASE_URL",
+      valueFrom = "arn:aws:ssm:${data.aws_region.current.name}:${data.aws_caller_identity.current.account_id}:parameter${module.database.admin_db_url_secret_name}"
+    },
+    {
+      name      = "POSTGRES_PASSWORD",
+      valueFrom = "arn:aws:ssm:${data.aws_region.current.name}:${data.aws_caller_identity.current.account_id}:parameter${module.database.admin_password_secret_name}"
+    },
+    {
+      name      = "POSTGRES_USER",
+      valueFrom = "arn:aws:ssm:${data.aws_region.current.name}:${data.aws_caller_identity.current.account_id}:parameter${module.database.admin_user_secret_name}"
+    }
+  ]
+  container_env_vars = [
+    {
+      name  = "NEXT_PUBLIC_DEMO_MODE",
```

probably shouldn't be named NEXT
```suggestion
      name  = "PRP_PUBLIC_DEMO_MODE",
```